src/Order/Base.php

```diff
@@ -144,11 +144,16 @@ public function get_shipping_options( $order ) {
 	 */
 	public function get_shipping_zone( $order ) {
 		$shipping_destination = $order->get_shipping_country();
+		$shipping_state       = $order->get_shipping_state();
 
 		if ( in_array( $shipping_destination, array( 'NL', 'BE' ) ) ) {
 			return $shipping_destination;
 		}
 
+		if ( in_array( $shipping_state, array( 'TF', 'GC' ) ) ) {
+			return 'ROW';
```

Comment:
`Grand Cape Mount` in Liberia have the same state code `GC`, we should check if the country is Spain as well.